library/pcs_cluster.py

```diff
@@ -78,6 +78,14 @@
     transport: 'udpu'
   run_once: true
 
+- name: Create cluster with redundant corosync links
+  pcs_cluster:
+    cluster_name: test-cluster
+    node_list: >
+      node1-eth0.example.com,node1-eth1.example.com
+      node2-eth0.example.com,node2-eth1.example.com
+    state: present
+
```

Comment:
```suggestion
  run_once: True
```
This is needed as in the other examples just in case that someone attempts to run this for all cluster nodes that should form one cluster.